### ISSUE TITLE
Sabbatical financing

### ISSUE THREAD

**Issue by ariddell** [pybee/paying-the-piper#58]
(Taking another approach to the idea proposed in #28 and #3)

**Goal**: Give an open source software developer three to six months of funding to work on a specific project/feature.

Three to six months is long enough for serious work to get done and not too long such that the developer couldn't return to the job/position they were working on before.

The sabbatical would be funded via a five-year zero-interest loan. The community/non-profit interested in supporting the developer would have to raise far less money (maybe $10-20k depending on the credit worthiness of the developer?). If there were a lot of developers interested in taking this deal, this approach could fund a lot of open source development. I'm assuming that a third-party, perhaps one of these new peer-to-peer lenders (e.g., https://www.upstart.com/), could handle the actual loan.

**Comment by freakboy3742** [comment]
An interesting idea - others have suggested sabbaticals, but this is definitely a new approach to funding that sabbatical. How do you envisage the loan being repaid? How does the $10-20k crowdfunded money relate to the income of a developer (which, ballpark, would be on the order of $30-$60k for an 6 months of an experienced developer)?

**Comment by ariddell** [comment]
Now that I think about it, I think $10-20k might fund 5 or more of these sabbatical loans.

The developers themselves would pay back the sabbatical loan over some period of time (e.g. 5 years). Depending on the developer's situation, I could imagine there would be some flexibility with the interest rate. For some a 0% interest loan would be fantastic, for others it might need to be something like -3%.

The loan would be for something like 75-100% salary replacement. I think you would leave this up to the developer and the loan provider (who would be taking the risk + the $10-20k).

Edit: add last paragraph

**Comment by nayafia** [comment]
Yes sabbaticals! Love the idea of funding 3-6 months of work. Why do this as a loan vs. as a grant? From a purely financial perspective, seems like debt/credit should only be taken on where there is financial upside, but here there is none.

Perhaps there's another way to structure this (mostly semantics), what if the developer "pledged"/"tithed" a small portion of their salary for the next year to pay for the sabbatical? 10% off $150k salary is something. Tithing to open source 😄

That being said, intuition and experience says make financing as uncomplicated as possible. Lending is a complex business (see how much Upstart has pivoted from its original idea). Another thing to look into could be a revolving fund structure for the nonprofit (has been used for energy efficiency, biotech research financing) - but again, generally implies a long-term financial upside.

Relatedly, I'm experimenting with ways to fund sabbaticals for developers: http://alphacollective.org and am down to talk through this idea more!

**Comment by ariddell** [comment]
The reason for doing loans vs. grants is that you could fund many more sabbaticals (maybe 5-10x as many?) for the same amount of money.

I would want to make it as simple as possible. The funder would tell the developer that they have this 0-percent loan guarantee (of up to 100%) of their salary to take 3-6 months off to do X. The developer can pick the size of the loan -- if they can live on 50% of their salary one would imagine most people would not take full salary replacement.

Good luck with alphacollective!

**Comment by freakboy3742** [comment]
@ariddell For me, the biggest problem with the loan approach is that it's still a financial burden on the person doing the work. As a philosophical question: If I'm someone producing open source software that is of value, why should _I_ have to pay for it's development? The fact that I should have to take out a loan so that BigCorp can generate bigger profits strikes me as the wrong approach.

Now, if expectation was that others would be paying off the loan for me, and the "loan" infrastructure is purely to get over the gap of "well, I'd like to fund this, but only if he delivers" - that's a different matter. It's still not an ideal situation, as it eliminates anyone who can't carry that financial burden - but it's better than the current completely unfunded situation.

**Comment by ariddell** [comment]
I think the zero or negative interest loan might appeal to a lot of developers who want to take time off to work intensively on an open source project. The chance to borrow, say, $20,000 to fund 3-6 months of work and pay it off at ~$350 a month over 5 years might appeal to a lot of the target group you're interested in supporting.

The prospect of supporting 10 developers this way rather than 1 (via an outright grant) appeals to me since I think it's likely there will not be any way to differentiate among all the top candidates for a sabbatical grant. I do like the spirit of Summer of Code (over, say, Ruby Together). This seems like a way to finance a "Summer of Code" for many open source developers in the US and Europe (where it seems like this loan might be arranged easily).

Of course if the developer lived in a country where the cost of living was much lower, it might be easy to do a partial grant (2 months paid, 1 month loan) or an outright grant.

edit: add last bit
